### PR TITLE
Add steps to run xDS in a non-exclusive project

### PR DESCRIPTION
**Pull request by lidizheng**
As title.

~~Though, I did encounter a problem with my manual run. Still investigating why the backend status didn't turn green.~~

```
======================================================================
ERROR: test_traffic_director_grpc_setup (__main__.BaselineTest) [6_add_server_backends_to_backend_service]
BaselineTest.test_traffic_director_grpc_setup
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/lidiz/grpc/tools/run_tests/xds_k8s_test_driver/tests/baseline_test.py", line 51, in test_traffic_director_grpc_setup
    self.setupServerBackends()
  File "/home/lidiz/grpc/tools/run_tests/xds_k8s_test_driver/framework/xds_k8s_testcase.py", line 145, in setupServerBackends
    self.td.wait_for_backends_healthy_status()
  File "/home/lidiz/grpc/tools/run_tests/xds_k8s_test_driver/framework/infrastructure/traffic_director.py", line 200, in wait_for_backends_healthy_status
    self.backends)
  File "/home/lidiz/grpc/tools/run_tests/xds_k8s_test_driver/framework/infrastructure/gcp/compute.py", line 284, in wait_for_backends_healthy_status
    _retry_backends_health()
  File "/home/lidiz/grpc/py37_native/lib/python3.7/site-packages/retrying.py", line 49, in wrapped_f
    return Retrying(*dargs, **dkw).call(f, *args, **kw)
  File "/home/lidiz/grpc/py37_native/lib/python3.7/site-packages/retrying.py", line 214, in call
    raise RetryError(attempt)
retrying.RetryError: RetryError[Attempts: 145, Value: False]

----------------------------------------------------------------------
Ran 1 test in 956.225s
```

---

The UHC failed to reach K8s pod because of firewall rules, it requires a special metadata tag that I wasn't aware of.

### CODE REVIEW

File: tools/run_tests/xds_k8s_test_driver/README.md [hunk 1]
```diff
@@ -89,8 +91,26 @@ python -m tests.security_test --helpful
...
+avoid if possible). Here is an example:
+
+```shell
+python3 -m tests.baseline_test \
   --flagfile="config/grpc-testing.cfg" \
   --kube_context="${KUBE_CONTEXT}" \
   --server_image="gcr.io/grpc-testing/xds-k8s-test-server-java:latest" \
   --client_image="gcr.io/grpc-testing/xds-k8s-test-client-java:latest" \
+  --namespace="box-$(date +"%F-%R")" \
+  --server_xds_port="$($(($RANDOM%1000 + 34567)))"
```
Comment (sergiitk) [inline tools/run_tests/xds_k8s_test_driver/README.md:115]:
```suggestion
  --server_xds_port="$(($RANDOM%1000 + 34567))"
```
Worked both in zsh and bash

File: tools/run_tests/xds_k8s_test_driver/README.md [hunk 2]
```diff
@@ -89,8 +91,26 @@ python -m tests.security_test --helpful
...
+be solved by supplying `--namespace` and `--server_xds_port`. The xDS port needs
+to be unique across the entire project (default port range is [8080, 8280],
+avoid if possible). Here is an example:
+
+```shell
+python3 -m tests.baseline_test \
   --flagfile="config/grpc-testing.cfg" \
   --kube_context="${KUBE_CONTEXT}" \
   --server_image="gcr.io/grpc-testing/xds-k8s-test-server-java:latest" \
   --client_image="gcr.io/grpc-testing/xds-k8s-test-client-java:latest" \
```
Comment (sergiitk) [inline tools/run_tests/xds_k8s_test_driver/README.md:113]: Could you update these too?

**Comment by lidizheng** [comment]
@sergiitk All requested changes done. PTALA.

**Review by sergiitk — APPROVED** [review]
Thank you!

**Pull request merged**